projects/ffmpeg/build.sh

```diff
@@ -121,30 +137,66 @@ rm $FFMPEG_DEPS_PATH/lib/*.so.*
 
 # Build ffmpeg.
 cd $SRC/ffmpeg
-PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
-    --cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
-    --extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
-    --extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
-    --prefix="$FFMPEG_DEPS_PATH" \
-    --pkg-config-flags="--static" \
-    --enable-ossfuzz \
-    --libfuzzer=$LIB_FUZZING_ENGINE \
-    --optflags=-O1 \
-    --enable-gpl \
-    --enable-libass \
-    --enable-libfdk-aac \
-    --enable-libfreetype \
-    --enable-libopus \
-    --enable-libtheora \
-    --enable-libvorbis \
-    --enable-libvpx \
-    --enable-libxml2 \
-    --enable-nonfree \
-    --disable-muxers \
-    --disable-protocols \
-    --disable-demuxer=rtp,rtsp,sdp \
-    --disable-devices \
-    --disable-shared
+if [[ "$ARCHITECTURE" == i386 ]]; then
+	PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
+		--cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
+		--extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
+		--extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
+		--prefix="$FFMPEG_DEPS_PATH" \
+		--pkg-config-flags="--static" \
+		--enable-ossfuzz \
+		--libfuzzer=$LIB_FUZZING_ENGINE \
+		--optflags=-O1 \
+		--enable-gpl \
+		--enable-nonfree \
+		--enable-libass \
+		--enable-libfdk-aac \
+		--enable-libfreetype \
+		--enable-libopus \
+		--enable-libtheora \
+		--enable-libvorbis \
+		--enable-libvpx \
+		--enable-libxml2 \
+		--enable-nonfree \
+		--disable-muxers \
+		--disable-protocols \
+		--disable-demuxer=rtp,rtsp,sdp \
+		--disable-devices \
+		--disable-shared \
+		--arch="i386" \
+		--cpu="i386" \
+		--disable-inline-asm \
+		--disable-asm \
+		--disable-neon \
+		;
+else
+	PKG_CONFIG_PATH="$FFMPEG_DEPS_PATH/lib/pkgconfig" ./configure \
+		--cc=$CC --cxx=$CXX --ld="$CXX $CXXFLAGS -std=c++11" \
+		--extra-cflags="-I$FFMPEG_DEPS_PATH/include" \
+		--extra-ldflags="-L$FFMPEG_DEPS_PATH/lib" \
+		--prefix="$FFMPEG_DEPS_PATH" \
+		--pkg-config-flags="--static" \
+		--enable-ossfuzz \
+		--libfuzzer=$LIB_FUZZING_ENGINE \
+		--optflags=-O1 \
+		--enable-gpl \
+		--enable-nonfree \
+		--enable-libass \
+		--enable-libfdk-aac \
+		--enable-libfreetype \
+		--enable-libopus \
+		--enable-libtheora \
+		--enable-libvorbis \
+		--enable-libvpx \
+		--enable-libxml2 \
+		--enable-nonfree \
+		--disable-muxers \
+		--disable-protocols \
+		--disable-demuxer=rtp,rtsp,sdp \
+		--disable-devices \
+		--disable-shared \
+		;
+fi
```

Comment:
The identical options should be factored out, that would also make this cleaner / easier to read